src/formatDate.js

```diff
@@ -50,7 +50,30 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const array = date.split(fromFormat[3]);
+  const newDate = [];
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    const lengthTrue = fromFormat[i].length !== toFormat[i].length;
```

Comment:
rename it, please
at least `true` in the name when `!==` in comparison sounds weird)